.github/workflows/check-catalyst.yaml

```
@@ -523,10 +523,7 @@ jobs:
 
     - name: Install additional dependencies (OpenQasm device)
       run: |
-        # TODO: Use the latest version of boto3 after fixing the issue with
-        # "Device SV1 does not support quantum circuits" from braket-pl-plugin.
-        # TODO: Pin amazon braket pennylane to a released version
-        pip install numpy amazon-braket-sdk git+https://github.com/amazon-braket/amazon-braket-pennylane-plugin-python.git@main "boto3==1.26"
+        pip install numpy amazon-braket-sdk amazon-braket-pennylane-plugin
```

This is thoroughly fixed in PR #345 
```suggestion
        # TODO: Use the latest version of boto3 after fixing the issue with
        # "Device SV1 does not support quantum circuits" from braket-pl-plugin.
        # TODO: Pin amazon braket pennylane to a released version
        pip install numpy amazon-braket-sdk git+https://github.com/amazon-braket/amazon-braket-pennylane-plugin-python.git@main "boto3==1.26"
```